tool/execution/linear/src/main/kotlin/flank/execution/linear/Transform.kt

```diff
@@ -1,4 +1,4 @@
-package flank.exection.linear
+package flank.execution.linear
```

Comment:
I'm renaming this entire package / corresponding directories. I expect this is a major api break. It seems like an odd spelling. Happy to drop...

Reply:
Let's just fix it while we're at it.